aipl/ops/sort.py

```diff
@@ -18,6 +18,21 @@ def op_grade_up(aipl, t:Table, *args):
     values = t.values
     return sorted(range(len(values)), key=values.__getitem__)
 
+@defop('incr', 1.5, 1.5)
+def op_incr(aipl, t:Table, step= 1, *args):
+    'Add column with incremental values'
```

Comment:
Let's make this documentation a bit more helpful, and include `step` and `base` in it.  This might be the only thing people read about !incr in the readme.  Maybe compare with iota from APL?